doc/netgen/LAUNCHPAD.md

```diff
@@ -0,0 +1,61 @@
+Netgen Site development with eZ Launchpad and Docker
+====================================================
+
+Software requirements
+---------------------
+
+* [Docker](https://docs.docker.com/) 
+* [eZ Launchpad](https://ezsystems.github.io/launchpad/)
+* [Yarn](https://yarnpkg.com/en/)
+
+Installation instructions
+-------------------------
+
+### Docker containers
+
+First create directory where Media site project will habitate and enter it:
+
+```bash
+mkdir my-project-with-media-suite
+cd my-project-with-media-suite
+```
+
+Instantiate media site with eZ Launchpad:
+
+```bash
+ez init netgen/media-site 1.1.3 netgen-media
```

Comment:
Is the version neccessary? It will go out of sync pretty soon and I'd rather not update it every time a new release is done?